src/Foundation/Maker.php

```diff
@@ -119,8 +119,9 @@ protected function getDependencies(array $parameters, array $primitives = [])
     {
         $dependencies = [];
 
+        /** @var ReflectionParameter $parameter */
         foreach ($parameters as $parameter) {
-            $dependency = $parameter->getClass();
+            $dependency = $parameter->getType();
```

Comment:
Did getType() exist in PHP 7.2?

Reply:
yes it is (available since 7.0): https://php.watch/versions/8.0/deprecated-reflectionparameter-methods

Reply:
Excellent. Thanks @Teranode!